inlong-manager/manager-web/src/main/resources/application-test.properties

```diff
@@ -63,3 +63,11 @@ es.auth.enable=false
 es.auth.user=admin
 # Elasticsearch password of authentication info
 es.auth.password=inlong
+
+# ClickHouse config
+# ClickHouse url
+ck.index.search.url=jdbc:clickhouse://127.0.0.1:8123/apache_inlong_audit
```

Comment:
Suggested changing to `audit.ck.jdbcUrl`, `audit.ck.username`, `audit.ck.password`.

Reply:
Done.